onnx/validate.py

```diff
@@ -0,0 +1,51 @@
+"""onnx validation
+
+This tool checks whether the models in a given set of files
+conform to the ONNX specification.
+"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import onnx
+from onnx import defs
+from onnx.defs import OpSchema
+import onnx.onnx_cpp2py_export.checker as c_checker
+import onnx.shape_inference
+
+import argparse
+
+
+def main():  # type: () -> None
```

Comment:
Please merge this file with [checker.py](https://github.com/onnx/onnx/blob/master/onnx/checker.py). It will cause a lot of confusion if we have one check and another validate.

Reply:
Sure. checker.py seemed structured to be more of an API, so I wasn't sure if it was OK to add a 'main' to it, but I agree that it's better to have just one entry.

Reply:
You can add the main here: 
https://github.com/onnx/onnx/blob/master/onnx/bin/checker.py